modules/ROOT/pages/index-policies.adoc

```diff
@@ -1,8 +1,25 @@
-= Security Policies
+= Edge Security Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-Anypoint Security features a high-performance, reliable, and scalable service to enforce policies that apply to all nodes deployed to your Runtime Fabric. +
+Anypoint Security features a high-performance, reliable, and scalable service to enforce security policies that apply to all nodes deployed to your Runtime Fabric. 
 
-You can protect your APIs using these policies:
+== Prerequisites
```

Comment:
I don't understand the topic hierarchy in this set. 
The topic set is titled Edge Security Policies, followed by a second-level topic that is Prerequisites (assumably prereqs for edge security?), followed by a peer second-level topic Security Policies for Edge (same as the first-level title) and another peer second-level prerequisites:

*Edge Security Policies
**Prerequisites (for edge security)
**Security Policies for Edge
**Prerequisites (for tokenization)

Reply:
These are two separate files (one is tokenization and one is security policies)